ros/api/v1/hosts.py

```diff
@@ -515,7 +515,11 @@ def get(self):
         ).count()
 
         non_psi_count = systems_with_performance_record_queryset.filter_by(psi_enabled=False).count()
-        psi_enabled_count = systems_with_performance_record_queryset.filter_by(psi_enabled=True).count()
+
+        psi_enabled_count = systems_with_performance_record_queryset.filter(
+            PerformanceProfile.operating_system['major'].astext.cast(Integer) != 7,
```

Comment:
FYI change would benefit from : https://github.com/RedHatInsights/ros-backend/pull/287#discussion_r1062291169

Reply:
Ack :+1:

Reply:
> @pgarciaq Hello Pau! It looks like we might need a RHEL 7 filter for `psi_enabled_count` as well. The reason for this is rules doesn't really have this _PSI cannot be enabled in RHEL 7_ logic.
> 
> One fine day if there is a bug from rules and we end up saving a bunch of `psi_enabled=True` RHEL 7 records, our psi numbers would look bad.
> 
> We can take this up with Xiangce but IMHO our code should be fault tolerant because it depends a lot on external sources of data.
> 
> Let me know your thoughts, thanks!
> 
> P.S. non_psi_count RHEL 7 exclusion changes are already in review and will be merged. I'll wait on your ack just for this one.

+1